cloudflare/resource_cloudflare_logpush_job_test.go

```diff
@@ -0,0 +1,73 @@
+package cloudflare
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+
+	"os"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccCloudflareLogpushJob_Basic(t *testing.T) {
+	zone := os.Getenv("CLOUDFLARE_DOMAIN")
+	rnd := acctest.RandString(10)
+	name := "cloudflare_logpush_job." + rnd
+	ownershipToken := os.Getenv("CLOUDFLARE_LOGPUSH_OWNERSHIP_TOKEN")
```

Comment:
We'll need to get this added to the Hashicorp CI setup to ensure we can run these acceptance tests, it works. You'll also want to add a new precheck method like I've previously done with [`testAccPreCheckAltDomain`](https://github.com/terraform-providers/terraform-provider-cloudflare/blob/master/cloudflare/provider_test.go#L47-L51) to halt tests if the value isn't present.

Reply:
Thanks, Jacob. I'll get this taken care of.

Reply:
Added a guard against a missing token after your example in 73cc6f6.

Opened Issue #288 to track this effort (as well as an internal ticket).